lib/runtime/wazero/instance.go

```diff
@@ -412,8 +422,19 @@ func NewInstance(code []byte, cfg Config) (instance *Instance, err error) {
 	logger.Info("instantiating a runtime!")
 	logger.Patch(log.SetLevel(cfg.LogLvl), log.SetCallerFunc(true))
 
+	// Prepare a cache directory.
+	cacheDir, err := os.MkdirTemp("", "wazeroCache")
```

Comment:
saves the `cacheDir` to delete it once we close the instance, looks like it does not remove the dir once it closes the cache https://github.com/tetratelabs/wazero/blob/253c0346e4c8d7bd604304447233c1a3ce0a71f7/cache.go#L70

Reply:
yeah exactly, I talked with @timwu20 and we were thinking we can just keep it for if we restart the node

Reply:
We should be putting this in `~/.gossamer` then if we plan to keep it.

Reply:
making this change, however, we have moved to using `xdg.DataHome` rather than `~/.gossamer` so using that instead